app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,22 @@
+.module {
+  width: auto;
+  max-width:100%;
+  background:
+  linear-gradient(
+  rgba(0, 0, 0, 0.7),
+  rgba(0, 0, 0, 0.7)
+  ),
+  url(asset-path("background.jpg"));
+  background-size: cover;
+  height: 100vh;
+}
+.page-title {
+  margin-top: 0;
```

Comment:
Properties should be ordered color, font-family, font-size, font-weight, margin-top